.gitignore

```diff
@@ -2,6 +2,7 @@ venv/
 __pycache__/
 ipynb_checkpoints
 glue/schema/*.parquet
+.pre-commit-config.yaml
```

Comment:
Add this local version of the `.pre-commit-config.yaml` to the git ignore. This allows the use to have control over their local hooks. Upon PR or push to main/master is when the factored out actions would take effect.